fuzzers/packfile_raw_fuzzer.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * libgit2 raw packfile fuzz target.
```

Comment:
oops, i just realized that i named the file wrong. it should have been `raw_packfile` in the filename for consistency with this comment.

the reason is that originally i wanted to split this into raw and cooked (with a valid hash at the end) packfiles, and it made more sense to name them `packfile_{raw,cooked}`. but then I figured out it was better to use the MSB of the first byte to change behavior, so I ended not using the `cooked` version.

Do you mind renaming this file to `raw_packfile_fuzzer`? or just `packfile_fuzzer` to avoid adding an unneeded qualifier to the name.

Reply:
Sure. I also prefer "packfile_fuzzer", so I've renamed it